packages/mobile/src/components/lineup-tile/CollectionTileTrackList.tsx

```diff
@@ -0,0 +1,147 @@
+import { LineupTrack } from 'audius-client/src/common/models/Track'
+import { range } from 'lodash'
+import { Pressable, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import Skeleton from 'app/components/skeleton'
+import { getPlayingUid } from 'app/store/audio/selectors'
+import { flexRowCentered, makeStyles } from 'app/styles'
+import { GestureResponderHandler } from 'app/types/gesture'
+
+// Max number of tracks to display
+const DISPLAY_TRACK_COUNT = 5
+
+type LineupTileTrackListProps = {
+  isLoading?: boolean
+  onPress: GestureResponderHandler
+  trackCount?: number
+  tracks: LineupTrack[]
+}
+
+const useStyles = makeStyles(({ palette, spacing, typography }) => ({
+  item: {
+    ...flexRowCentered(),
+    width: '100%',
```

Comment:
been trying to figure out myself here, should we do width: 100% or flex 1?

Reply:
Either should be fine, with flex 1 we would always need to specify flex direction row I guess